excavator/utils.py

```diff
@@ -80,7 +81,48 @@ def env_int(name, required=False, default=empty):
     value = get_env_value(name, required=required, default=default)
     if value is empty:
         raise ValueError(
-            "`env_int` requires either a default value to be specified, or, for "
+            "`env_int` requires either a default value to be specified, or for "
             "the variable to be present in the environment"
         )
     return int(value)
+
+
+def env_timestamp(name, default=empty, required=False):
+    if required and default is not empty:
+        raise ValueError("Using `default` with `required=True` is invalid")
+
+    value = get_env_value(name, required=required, default=empty)
+    # change datetime.datetime to time, return time.struct_time type
+    if default is not empty and value is empty:
+        return default
+    if value is empty:
+        raise ValueError(
+            "`env_timestamp` requires either a default value to be specified, or "
+            "for the variable to be present in the environment"
+        )
+
+    timestamp = float(value)
+    return datetime.datetime.fromtimestamp(timestamp)
```

Comment:
Murr, i'm not really sure if that's a good thing either...

```python
In [7]: datetime.datetime.utcnow()
Out[7]: datetime.datetime(2015, 3, 11, 17, 21, 18, 382463)

In [8]: datetime.datetime.now()
Out[8]: datetime.datetime(2015, 3, 11, 11, 21, 20, 132012)
```

Also, without `pytz` how do you do timezones?